fxa/tests/test_oauth.py

```diff
@@ -602,6 +615,71 @@ def test_monkey_patch_for_gevent(self):
         fxa._utils.requests = old_requests
 
 
+class TestJwtToken(unittest.TestCase):
+
+    server_url = TEST_SERVER_URL
+
+    def callback(self, request):
+        if self.verify_success:
+            return (200, {}, self.body)
+        return (500, {}, "{}")
+
+    def setUp(self):
+        self.client = Client(server_url=self.server_url)
+        self.body = ('{"user": "alice", "scope": ["profile"],'
+                     '"client_id": "abc"}')
+        responses.add_callback(responses.POST,
+                               'https://server/v1/verify',
+                               callback=self.callback,
+                               content_type='application/json')
+        add_jwks_response()
+        self.verify_success = True
+
+    def get_file_contents(self, filename):
+        return jwt.algorithms.RSAAlgorithm.from_jwk(open(
+            os.path.join(
+                os.path.dirname(__file__),
+                filename
+            )
+        ).read())
+
+    @responses.activate
+    def test_good_jwt_token(self):
+        private_key = self.get_file_contents("private-key.json")
+        result = str(jwt.encode({}, private_key, "RS256", {"typ": "at+jwt"}))
+        self.client.verify_token(result)
+
+    @responses.activate
+    def test_wrong_key_jwt_token(self):
+        self.verify_success = False
+        bad_key = self.get_file_contents("bad-key.json")
+        result = jwt.encode({}, bad_key, "RS256", {"typ": "at+jwt"})
+        try:
+            self.client.verify_token(result)
+        except Exception:
+            return
+        raise Exception("verifying the token signed with the wrong key did not cause an error.")
```

Comment:
micro-nit: when I first read the error string here I was confused, because I parsed "did not cause an error" as a statement about the expected behaviour rather than about the *violation* of expected behaviour. Could you please re-word this to something like "failed to cause an error" or "should have caused an error" or similar, for clarity?